avocado/plugins/spawners/process.py

```diff
@@ -2,8 +2,13 @@
 import os
 import sys
 
+from avocado.core.dependencies.requirements import cache
 from avocado.core.plugin_interfaces import Spawner
 from avocado.core.spawners.common import SpawnerMixin, SpawnMethod
+from avocado.core.teststatus import STATUSES_NOT_OK
+
+ENVIRONMENT_TYPE = 'local'
+ENVIRONMENT = 'localhost.localdomain'
```

Comment:
I am not sure if this is the best choice for the environment name.  Maybe use the actual hostname?